Android/MvvmSeed/app/src/test/java/com/makingsense/android/mvvmseed/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.makingsense.android.mvvmseed;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() throws Exception {
```

Comment:
What do you mean? my unit test is not good enough for you? 😄 

Yep, this is just the initial setup of the project, 99% are required project files and standard code. Real unit test coming soon..